src/sidebar/test/service-url-test.js

```diff
@@ -1,20 +1,166 @@
 'use strict';
 
-var serviceUrl = require('../service-url');
+var proxyquire = require('proxyquire');
```

Comment:
Some comment as with `service-url.js`, the diff is messed up so I recommend just viewing the `service-url-test.js` file (which is completely new) instead of looking at the diff